lib/factory/EntryFieldDescription.js

```diff
@@ -11,11 +28,18 @@ var MARKDOWN_LINK_REGEX = /\[([^)]+)\]\(([^\]]+)\)/g;
  * @return {String}
  */
 function linkify(text) {
-  return text.replace(MARKDOWN_LINK_REGEX, '<a href="$2" target="_blank">$1</a>');
-}
+  return text.replace(/\[([^)]+)\]\((https?:\/\/[^\]]+)\)/g, function(_, text, link) {
```

Comment:
RegEx updated to link only links instead of `<a href="javascript:alert(1)">link</a>`.